include/odp/api/spec/packet_io.h

```diff
@@ -665,7 +665,8 @@ int odp_pktio_mac_addr_set(odp_pktio_t pktio, const void *mac_addr,
  * @param pktio        Ingress port pktio handle.
  * @param default_cos  Class-of-service set to all packets arriving at this
  *                     ingress port, unless overridden by subsequent
- *                     header-based filters.
+ *                     header-based filters. Use ODP_COS_INVALID to remove
+ *                     the default CoS.
```

Comment:
"... unless overridden by subsequent header-based filters. " refers to deprecated API functions. I'd suggest to remove the sentence now.

Reply:
v4.